assets/js/tablegenerator.js

```diff
@@ -20,37 +26,45 @@ async function getClassList() {
             headers: {
                 "content-type": "application/json",
             },
-        });
+        })
 
+        // Error check
         if (!response.ok) {
-            throw new Error('Network response was not ok');
+            throw new Error('Network response was not ok')
```

Comment:
We should probably do this for all of our code, but whenever we throw an error you should include the status. Something more like this:

throw new Error(`HTTP error! Status: ${response.status}`);